src/import-data/ImportData.ts

```diff
@@ -107,10 +107,20 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
     if (workspace.cloudPlatform === 'Azure') {
       // find wds for this workspace
       const wdsUrl = await Ajax().Apps.listAppsV2(workspace.workspaceId).then(resolveWdsUrl);
-      const wdsDataTableProvider = new WdsDataTableProvider(workspace.workspaceId, wdsUrl);
+      const { namespace, name } = workspace;
 
       // call import snapshot
-      await wdsDataTableProvider.importTdr(workspace.workspaceId, importRequest.manifestUrl);
+      const { jobId } = await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
+        url: importRequest.manifestUrl.toString(),
```

Comment:
Can this arg be changed to type `URL`? I guess maybe that would propagate back to `ListAppItem.proxyUrls`; if so it would be a bigger change. But the type should probably be changed to URL at some point.